### PR TITLE
Dark Mode : Gas fee

### PR DESCRIPTION
**Pull request by GuillaumeRx**

### CODE REVIEW

File: ui/components/app/gas-details-item/index.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .gas-details-item {
   &__gas-fee-warning {
-    color: var(--secondary-1);
+    color: var(--color-warning-default);
```
Comment (georgewrmarshall) [inline ui/components/app/gas-details-item/index.scss:3]: Hmm unfortunately I think we may have to break some semantic rules here. 😅 
```suggestion
    color: var(--color-secondary-default);
```

Our warning color isn't really visible as text. Going to make a note and work towards fixing this. You used the token correctly. The token needs updating but that will take time.

Screen Shot 2022-03-16 at 3 12 38 PM

File: ui/components/app/gas-timing/index.scss [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 
   &--positive-V2 {
-    color: var(--success-3);
+    color: var(--color-success-default);
     font-weight: bold;
   }
 
   &--warning {
-    color: var(--alert-3);
+    color: var(--color-warning-default);
```
Comment (georgewrmarshall) [inline ui/components/app/gas-timing/index.scss:14]: Ahh this kills me... 
```suggestion
    color: var(--color-secondary-default);
```

File: ui/components/app/gas-timing/index.scss [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 
   &--negative {
-    color: var(--error-1);
+    color: var(--color-error-default);
     font-weight: bold;
   }
 
   &--negative-V2 {
-    color: var(--secondary-1);
+    color: var(--color-warning-default);
```
Comment (georgewrmarshall) [inline ui/components/app/gas-timing/index.scss:23]:
```suggestion
    color: var(--color-secondary-default);
```

**Review by georgewrmarshall — COMMENTED** [review]
Looking good! Made some suggestions

File: ui/components/app/gas-timing/index.scss (hunk 2 above)
Comment (GuillaumeRx) [reply ui/components/app/gas-timing/index.scss:14]: Yeah for all of those warning colors I was divided between using warning and secondary, I don't know how we could define which to use where

File: ui/components/app/gas-customization/advanced-gas-inputs/index.scss [hunk 4]
```diff
@@ -98,8 +98,8 @@
       }
 
       &__i-wrap:hover {
-        background: #4eade7;
-        color: var(--white);
+        background: var(--color-primary-muted);
+        color: var(--color-background-default);
```
[the remark this answers is not in the document]
Comment (GuillaumeRx) [reply ui/components/app/gas-customization/advanced-gas-inputs/index.scss:102]: Nice catch ! I always forget those inverse colors

**Comment by darkwing** [comment]
Should these be part of this PR or am I misreading the scope of this? 
PR

**Comment by GuillaumeRx** [comment]
> Should these be part of this PR or am I misreading the scope of this? PR

@darkwing the top part should be fixed by https://github.com/MetaMask/metamask-extension/pull/13932 since it updates the `Popover` component

**Comment by GuillaumeRx** [comment]
`radio-group` fixed @darkwing  :)

**Comment by darkwing** [comment]
Just a few things I spotted when I turned on Advanced Gas Estimation in the settings:

hoverinfo
GasTooltip
OptionHover

**Review by georgewrmarshall — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**